core/build.gradle

```diff
@@ -3,6 +3,7 @@ apply plugin: 'bintray-release'
 
 dependencies {
     compile 'com.android.support:support-v4:23.0.1'
+    compile 'io.reactivex:rxandroid:1.0.1'
```

Comment:
everyone that pulls in this lib will also pull in rxandroid? Is there a provided scope - presumably if people are wanting to use the RxJava bindings, they'll have this dependency anyway?